test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -550,26 +550,6 @@ static public void testCreateArrayFlattenedLine2D() throws Throwable {
 		assertEquals(getX.invoke(getFlatEn.invoke(line2D_2_check)), getX.invoke(getFlatEn.invoke(line2D_2)));
 		assertEquals(getY.invoke(getFlatEn.invoke(line2D_1_check)), getY.invoke(getFlatEn.invoke(line2D_1)));
 		assertEquals(getY.invoke(getFlatEn.invoke(line2D_2_check)), getY.invoke(getFlatEn.invoke(line2D_2)));
-	}	
-	
-	/*
-	 * Test defaultValue with ref type
-	 *
-	 * class DefaultValueWithNoneValueType {
-	 * 	Object f1;
-	 * 	Object f1;
-	 * }
-	 *
-	 */
-	@Test(enabled=false, priority=3)
-	static public void testDefaultValueWithNonValueType() throws Throwable {
-		String[] fields = {"f1:Ljava/lang/Object;:NR", "f2:Ljava/lang/Object;:NR"};
-		Class<?> defaultValueWithNonValueType = ValueTypeGenerator.generateRefClass("DefaultValueWithNonValueType", fields);
```

Comment:
I think there are differences between this and `testNullRestrictedFieldWhereImplicitCreationHasNoDefaultFlag()`. Here is generating a reference class while `testNullRestrictedFieldWhereImplicitCreationHasNoDefaultFlag()` is generating a value class.

Reply:
You're right. In that case I think its covered by ValhallaAttributeTests.testNonValueTypeClassWithImplicitCreationAttribute. The ImplicitCreation attribute which would contain the flag to enable default values is not allowed in a reference class so this would fail during class verification.

Reply:
>  In that case I think its covered by ValhallaAttributeTests.testNonValueTypeClassWithImplicitCreationAttribute.

Could you update your commit message to mention testNonValueTypeClassWithImplicitCreationAttribute ?